### ISSUE TITLE
Issue with Path set up for Spark on Windows 10

### ISSUE THREAD

**Issue by circushaase** [mGalarnyk/Installations_Mac_Ubuntu_Windows#6]
I am having issues with my path in setting up Spark on my Windows 10 machine. I'm pretty sure it is not a security issue.

In your video, you add the spark path to your Anaconda path. I do not have Anaconda because it would not install/set paths properly (I’ve had several people look at this and not figure out the problem), so I have Miniconda installed instead. 

When I open my Environmental Variables and click to Edit my “Path” a window with what looks like many paths opens up, but none of them are Miniconda. I tried just adding a new path for the spark path, but this doesn’t seem to work. I've attached a screenshot.

![path_issue](https://user-images.githubusercontent.com/24961288/36116727-e34ae3ac-0ffc-11e8-97e9-d142d34cad78.PNG)

**Comment by circushaase** [comment]
Got it fixed! 
The issue was that I was copying the commands from the article, but my version of Spark is newer, so I had the wrong version number typed in my path. I realized I was doing this, but only about halfway through the process! When I was trying to fix my path, I was looking under “System Variables”, but the path that needed editing was under “User Variables”. Once I fixed the version number here, all was fixed (it seems).
